XamlControlsGallery/DataModel/ControlInfoData.json

```diff
@@ -1957,7 +1958,7 @@
               "Uri": "https://www.docs.microsoft.com/uwp/api/microsoft.ui.xaml.controls.animatedicon"
             },
             {
-              "Title": "AnimatedIcon - Guidance",
+              "Title": "AnimatedIcon - Guidelines ",
```

Comment:
For consistency, should this just be "Guidelines"?

Reply:
Good point, but should it be reversed? Other links seem to follow that format, eg "AnimatedIcon - API".

Reply:
It seems that all controls just have "Guidelines" while the other links, such as the API links, are name "[control name] API" etc.

Reply:
I just took a quick look through the gallery and I think most pages have just "Guidelines", so leaving it at that would probably be most consistent. But your point is valid that there's inconsistency between the formatted API link and guidance link. That might be a good thing to implement as a larger change.